homeassistant/components/rest/sensor.py

```diff
@@ -224,7 +236,15 @@ class RestData:
     """Class for handling the data retrieval."""
 
     def __init__(
-        self, method, resource, auth, headers, data, verify_ssl, timeout=DEFAULT_TIMEOUT
+        self,
+        method,
+        resource,
+        auth,
+        headers,
+        data,
+        verify_ssl,
+        client_certificate_path=None,
```

Comment:
Why are you setting this to `None`?

Reply:
there are other components that use ``RestData`` so i need to set a default value for those that do not pass the parameter

Reply:
Äähh, I forgot that we allowed that in the past.

Reply:
so, will it be merged or are there additional changes needed?